java-engine/src/main/java/messaging/ContextMessage.java

```diff
@@ -92,8 +92,6 @@ public static int endContextMessage(FlatBufferBuilder builder) {
     int o = builder.endTable();
     return o;
   }
-  public static void finishContextMessageBuffer(FlatBufferBuilder builder, int offset) { builder.finish(offset); }
-  public static void finishSizePrefixedContextMessageBuffer(FlatBufferBuilder builder, int offset) { builder.finishSizePrefixed(offset); }
```

Comment:
fallout from running flatc

Reply:
Yeah okay that's weird. I guess it's okay if the tests are green, at least for now